services/productPicServices.js

```diff
@@ -0,0 +1,9 @@
+import { productPicDao } from '../models';
+
+const productPic = async category => {
+  const pic = await productPicDao.productPic();
+
+  return pic;
```

Comment:
```suggestion
  return await productPicDao.productPic();
```

dao단에서 하신것 처럼 한줄의 리턴문으로 하셔도 좋을거같아요~